Description: Fix cross-compilation
 Don't forcibly -I/usr/include during the build process, the compiler
 already knows how to look in that directory.
 .
Author: Kyle Moffett <Kyle.D.Moffett@boeing.com>

---
Forwarded: no
Last-Update: 2011-10-13

--- a/src/Makefile
+++ b/src/Makefile
@@ -68,7 +68,7 @@ CFLAGS ?= -O -Wall -W -Wundef -Wformat-y
 
 PCRE_LDFLAGS ?= -lpcre
 
-override CFLAGS += -I../include -I$(INCLUDEDIR) -D_GNU_SOURCE $(DISABLE_FLAGS) $(PCRE_CFLAGS)
+override CFLAGS += -I../include -D_GNU_SOURCE $(DISABLE_FLAGS) $(PCRE_CFLAGS)
 
 SWIG_CFLAGS += -Wno-error -Wno-unused-variable -Wno-unused-but-set-variable -Wno-unused-parameter \
 		-Wno-shadow -Wno-uninitialized -Wno-missing-prototypes -Wno-missing-declarations
